packages/eclipse/src/components/banner.tsx

```
@@ -113,14 +114,42 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
       >
+        {changeLayout && open ? (
+          <style>
+            {globalKey
+              ? `:root:not(.${globalKey}) { --fd-banner-height: ${height ?? "3rem"}; }`
+              : `:root { --fd-banner-height: ${height ?? "3rem"}; }`}
+          </style>
+        ) : null}
+        {globalKey ? <style>{`.${globalKey} #${id} { display: none; }`}</style> : null}
+        {globalKey ? (
+          <script
+            dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)